lib/kafo/execution_environment.rb

```diff
@@ -48,6 +52,7 @@ def configure_puppet(settings = {})
         'hiera_config'    => hiera_config,
         'reports'         => 'store',
         'reportdir'       => reportdir,
+        'libdir'          => libdir,
```

Comment:
Nit: I tried to keep the settings sorted

Reply:
given `reports` should sort after `reportdir` I'm not sure you succeeded, but I did a `:sort`.